index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Software Developer's Conference</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="Design/css/style.css">
+  </head>
+  <body>
+    <header class = "shared_header">
+      <!--Dynamic header content loaded from JS DOM-->
+    </header>
+
+    <main class="parent_main">
+       <section class="container-fluid jumbotron slider intro_header pr-5">
+          <div class="row">
+           <div class="intro col-sm-12 mt-5">
+            <h3 class = "hello_intro orange text-md-left text-center">"Home for Creatives!"</h3>
+            <h1 class="text-img-effect font-weight-bold font-style-medium orange text-md-left text-center">SOFTWARE DEVELOPERS</h1>
+            <h1 class="text-img-effect font-weight-bold orange text-md-left text-center">ETHIOPIA SUMMIT</h1>
+            <h1 class="text-img-effect font-weight-bold orange text-md-left text-center">2022</h1>
+            <p class="p-4 intro_description text-dark">
+              Creative Hub Ethiopia supports creative innovators, designers and SMEs 
+              in various fields by deepening understanding of global design and industrial concepts, 
+              providing state-of-the-art working tools and resources, and enhancing the link between young innovators, 
+              industrial and governmental entities.
+            </p>
+            <h2 class="font-weight-bold mb-3">2022.01.07(MON) ~ 12(SAT)</h2>
+            <p>@ Creative-Hub Ethiopia</p>
+           </div>
+          </div>
+       </section>
+
+      <section class="container-fluid main_program" id="Program">
```

Comment:
- [ ] It looks like there is a problem with one of your items here in this section. It seems that the first item has not the proper size and is not being properly displayed on large screens. Take a look at the image below for reference:
![Screenshot_2022-02-09_19-28-28](https://user-images.githubusercontent.com/59100477/153266353-f6de347e-5342-47d6-afe4-bd5fe616a9fb.png)

Reply:
Done.